libmscore/sym.cpp

```diff
@@ -6675,6 +6670,17 @@ void ScoreFont::scanUserFonts(const QString& path, bool system)
 
       _allScoreFonts = _builtinScoreFonts;
       _allScoreFonts << _userScoreFonts << _systemScoreFonts;
+
```

Comment:
Should also occur when SMuFL user directory changes because: 
```cpp
  QString userFontsPath = preferences.getString(PREF_APP_PATHS_MYSCOREFONTS);
      scanUserFonts(userFontsPath);
      preferences.addOnSetListener([](const QString& key, const QVariant& value) {
            if (key == PREF_APP_PATHS_MYSCOREFONTS)
                  scanUserFonts(value.toString());
            });
```
above triggers scanUserFonts again

Reply:
So all is fine...